.circleci/config.yml

```diff
@@ -19,16 +19,21 @@ anchors:
 
   java_defaults: &java_defaults
     <<: *defaults
+    resource_class: medium+
     environment:
       # As best I can tell (dmohs, 7 Feb '17), this is the only way to set a memory limit that Java
       # processes executed within CircleCI's docker containers will respect. Very helpful resource:
       # https://circleci.com/blog/how-to-handle-java-oom-errors/
       #
       # In Feb 2017, this was set to 3G. But in Feb 2019 (RW-2194) we started seeing OOM errors,
       # so we bumped this down further to 2G.
-      JAVA_TOOL_OPTIONS: -Xmx2g
+      # JAVA_TOOL_OPTIONS: -Xmx2g
```

Comment:
I don't think reducing memory size for Gradle is the right approach. Gradle needs decent size of memory to be "happy" and "fast". Reducing it is not what we should do. Instead, use a larger container `medium+` (default was `medium`) and assign gradle a larger memory size. So far, all my testing, I have not had an issue with OOM anymore.

Reply:
The counter argument is that we should not need a large (and more expensive) machine to run our unit tests. Have you experimented with changing the resource class independent of the other changes you've made here? If this one difference gets us from 15m -> 6m, great. If it has no effect on latency, then leaving it at the default seems preferred.

Please update the comment here and remove commented out code.